packages/notebook-extension/schema/tracker.json

```diff
@@ -366,6 +366,16 @@
     ]
   },
   "jupyter.lab.shortcuts": [
+    {
+      "command": "notebook:access-last-history",
+      "keys": ["Alt ArrowUp"],
```

Comment:
Let's move these down to all other commands with `.jp-Notebook.jp-mod-editMode` selector?